Worktrolly/worktrolly
# Dev/org page

### Functionality:
To make Organization & Team component , so that all organizations can use WorkTrolly.

### Solution:

- By Making CreateNewOrganization, OrgDetails, TeamDetails Components and Modified Login Component for users to Sign Up  using Email.
- By Implementing Backend of CreateNewOrg to store the Organization details in the Backend.
- Making Security Keys in Backend.

### Risk level:

### How to test:

- To get no errors at the backend. Update Node in your local system.
- Open a terminal
- Update Node with **sudo npm i -g n**  , **sudo n stable**
- Close the terminal . And open new Terminal . Check node version using **node -v** . It should be v14.15.* .
- ng serve
- firebase emulators:start
- Click Login and Login with Google & then Logout.
- Click Login and Click on **Create an Organization** Button.
